docs/AutomaticMocking.md

```diff
@@ -26,49 +31,41 @@ module.exports = {
 
 ```javascript
 // login.js
-var CurrentUser = require('./CurrentUser.js');
+const CurrentUser = require('./CurrentUser.js');
 ```
 
-If we run `login.js` with node, Jest will not become involved at all and the program will execute as you'd normally expect. However, if you run a unit test for the `login.js` file, Jest takes over and modifies `require()` such that the code behaves in the following way:
+If we run `login.js` with node, Jest will not become involved at all and the
+program will execute as you'd normally expect. However, if you run a unit test
+for the `login.js` file, Jest takes over and modifies `require()` such that the
+code behaves in the following way:
 
 ```javascript
-var CurrentUser = {
-  getID: jest.genMockFunction(),
-  setID: jest.genMockFunction()
+const CurrentUser = {
+  getID: jest.fn(),
+  setID: jest.fn()
 };
 ```
 
-With this setup, you cannot accidentally rely on the implementation details of `CurrentUser.js` when testing `login.js` because all of the calls to the `CurrentUser` module are mocked. Additionally, testing becomes easier in practice because you don't have to write any boilerplate in your tests to setup mock objects for the dependencies you don't want to test.
+With this setup, you cannot accidentally rely on the implementation details of
+`CurrentUser.js` when testing `login.js` because all of the calls to the
+`CurrentUser` module are mocked. Additionally, testing becomes easier in
+practice because you don't have to write any boilerplate in your tests to
+setup mock objects for the dependencies you don't want to test.
 
 How does it work?
 -----------------
 
-Jest actually implements its own version of the `require()` function in the testing environment. Jest's custom `require()` function loads the real module, inspects what it looks like, and then makes a mocked version based on what it saw and returns that.
+Jest actually implements its own version of the `require()` function in the
+testing environment. Jest's custom `require()` function loads the real module,
+inspects what it looks like, and then makes a mocked version based on what it
+saw and returns that.
 
-This means Jest is going to give you an object with the same shape as the real module, but with mocks for each of the exported values instead of the real values.
+This means Jest is going to give you an object with the same shape as the real
+module, but with mocks for each of the exported values instead of the real
+values.
 
-```javascript
-// Single function
-jest.genMockFromModule(function() { /* ... */ })
-  -> jest.genMockFunction();
-
-// Object
-jest.genMockFromModule({
```

Comment:
I thought this was confusing because `genMockFromModule` takes a module name, not an actual module exports (I don't think it ever did). My plan is to make the moduleMocker a separate package with the next release of Jest and document the mocking part completely separately.